target/linux/ath79/dts/qca9558_airtight_c-65.dts

```diff
@@ -32,7 +32,7 @@
 		power_amber: power_amber {
 			label = "c-65:amber:power";
 			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
-			default-state = "on";
+			default-state = "off";
```

Comment:
Remove this property, 'off' is the default state, no need to set it explicitly again.

Reply:
As this is power LED by default it should be 'on', and left to user to configure it to desired state in `/etc/config/system`.

Reply:
It is "off" to match the others, the user can change everything to his taste afterward.

Reply:
Property wiped.